src/hypervisor.c

```diff
@@ -107,6 +108,43 @@ cc_oci_expand_net_device_cmdline(struct cc_oci_config *config, guint index) {
 	return g_strdup("");
 }
 
+/*
+ *
+ *
+ */
+#define DPDK_CMDLINE_OBJ "memory-backend-file,id=dpdkmem,size=2048M,mem-path=/dev/hugepages,share=on,prealloc=on"
```

Comment:
I think the @VM_MEM@ and @VM_MEM_MAX@, with hardwired defaults in the runtime and command line options for over-ride feels right for now. Some notes:
 - longer term we will want to take into account the orchestration supplied 'memory_limit_in_bytes' and other information passed in to us to adjust the value if requested etc.
 - The 1G disparity between VM_MEM and VM_MEM_MAX is currently the extra 1G 'slot' for us to map the nvdimm 'image' into - so we may need to sanity check the passed in command line args to ensure we have enough space for the image as well.